app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -480,7 +480,10 @@ class HomeFragment : Fragment() {
         val profilerStartTime = requireComponents.core.engine.profiler?.getProfilerTime()
 
         super.onViewCreated(view, savedInstanceState)
-        context?.metrics?.track(Event.HomeScreenDisplayed)
+        context?.metrics?.apply {
+            track(Event.HomeScreenDisplayed)
```

Comment:
Should we remove this then? since we will now have a counter?

Reply:
I wouldn't think so since the event should be, in theory, used to follow a sequence of user interactions, whereas the counter will be used for some quantitative calculations and comparisons.